<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWidget</class>
 <widget class="QWidget" name="LoginWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>登录界面</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/resource/background/app.png</normaloff>:/resource/background/app.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#page, QWidget#page_2 {
	border: 1px solid #A5A6A8;
	border-radius: 4px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>50</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>50</number>
   </property>
   <item>
    <widget class="RotatingStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>1</number>
       </property>
       <item>
        <widget class="QWidget" name="widgetWinTitle" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1,0,0,0">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="labelWinIcon">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="images.qrc">:/resource/images/robot.png</pixmap>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelWinTitle">
            <property name="styleSheet">
             <string notr="true">color: #FFFFFF;</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnWinMenu">
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="images.qrc">
              <normaloff>:/resource/common/ic_login_cfg.png</normaloff>:/resource/common/ic_login_cfg.png</iconset>
            </property>
            <property name="autoRepeatDelay">
             <number>400</number>
            </property>
            <property name="autoRepeatInterval">
             <number>150</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnWinMin">
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="images.qrc">
              <normaloff>:/resource/common/ic_min.png</normaloff>:/resource/common/ic_min.png</iconset>
            </property>
            <property name="autoRepeatDelay">
             <number>400</number>
            </property>
            <property name="autoRepeatInterval">
             <number>150</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnWinClose">
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="images.qrc">
              <normaloff>:/resource/common/ic_close_white.png</normaloff>:/resource/common/ic_close_white.png</iconset>
            </property>
            <property name="autoRepeatDelay">
             <number>400</number>
            </property>
            <property name="autoRepeatInterval">
             <number>150</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widgetBody" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="labelWeclcom">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="images.qrc">:/resource/background/logo.jpg</pixmap>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="spacing">
             <number>10</number>
            </property>
            <property name="leftMargin">
             <number>40</number>
            </property>
            <property name="topMargin">
             <number>20</number>
            </property>
            <property name="rightMargin">
             <number>40</number>
            </property>
            <property name="bottomMargin">
             <number>20</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <property name="spacing">
               <number>15</number>
              </property>
              <item>
               <widget class="QLabel" name="labelUserHead">
                <property name="minimumSize">
                 <size>
                  <width>82</width>
                  <height>82</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>82</width>
                  <height>82</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">QLabel#labelUserHead {
	border: 1px solid #C9C9C9;
}</string>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="pixmap">
                 <pixmap resource="images.qrc">:/resource/head/head-64.png</pixmap>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="widgetInput" native="true">
                <layout class="QVBoxLayout" name="verticalLayout_3">
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QIconLineEdit" name="lineEditUser">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>25</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>25</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>xiaoni</string>
                   </property>
                   <property name="placeholderText">
                    <string>用户名</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QIconLineEdit" name="lineEditPasswd">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>25</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>25</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>1234</string>
                   </property>
                   <property name="echoMode">
                    <enum>QLineEdit::Password</enum>
                   </property>
                   <property name="placeholderText">
                    <string>用户密码</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout">
                   <item>
                    <widget class="QCheckBox" name="checkBoxPasswd">
                     <property name="text">
                      <string>记住密码</string>
                     </property>
                     <property name="checked">
                      <bool>true</bool>
                     </property>
                     <property name="autoRepeatDelay">
                      <number>400</number>
                     </property>
                     <property name="autoRepeatInterval">
                      <number>150</number>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QCheckBox" name="checkBoxAutoLogin">
                     <property name="text">
                      <string>自动登录</string>
                     </property>
                     <property name="checked">
                      <bool>false</bool>
                     </property>
                     <property name="autoRepeatDelay">
                      <number>400</number>
                     </property>
                     <property name="autoRepeatInterval">
                      <number>150</number>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <property name="spacing">
               <number>15</number>
              </property>
              <item>
               <widget class="QPushButton" name="btnLogin">
                <property name="focusPolicy">
                 <enum>Qt::NoFocus</enum>
                </property>
                <property name="toolTip">
                 <string>登录</string>
                </property>
                <property name="text">
                 <string>登录</string>
                </property>
                <property name="autoRepeatDelay">
                 <number>400</number>
                </property>
                <property name="autoRepeatInterval">
                 <number>150</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>1</number>
       </property>
       <item>
        <widget class="QWidget" name="widgetWinTitle_2" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,0,0">
          <property name="spacing">
           <number>5</number>
          </property>
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="labelWinTitle_2">
            <property name="styleSheet">
             <string notr="true">color: #FFFFFF;</string>
            </property>
            <property name="text">
             <string>登录配置</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnWinMin_2">
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="images.qrc">
              <normaloff>:/resource/common/ic_min.png</normaloff>:/resource/common/ic_min.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnWinClose_2">
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="images.qrc">
              <normaloff>:/resource/common/ic_close_white.png</normaloff>:/resource/common/ic_close_white.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widgetBody_2" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>服务器设置</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <property name="rightMargin">
                <number>0</number>
               </property>
               <item>
                <layout class="QFormLayout" name="formLayout">
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_3">
                   <property name="text">
                    <string>服务器地址配置：</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QIPLineEdit" name="lineEditHostAddr">
                   <property name="text">
                    <string>192.168.5.181</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="label_4">
                   <property name="text">
                    <string>消息服务器端口：</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QLineEdit" name="lineEditHostMsgPort">
                   <property name="text">
                    <string>60001</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0">
                  <widget class="QLabel" name="label_5">
                   <property name="text">
                    <string>文件服务器端口：</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="1">
                  <widget class="QLineEdit" name="lineEditHostFilePort">
                   <property name="text">
                    <string>60002</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>157</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btnCancel">
              <property name="text">
               <string>取消</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnSaveCfg">
              <property name="text">
               <string>确定</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QIPLineEdit</class>
   <extends>QLineEdit</extends>
   <header>clineedit.h</header>
  </customwidget>
  <customwidget>
   <class>RotatingStackedWidget</class>
   <extends>QStackedWidget</extends>
   <header>rotatingstackedwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QIconLineEdit</class>
   <extends>QLineEdit</extends>
   <header>clineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnWinMin_2</sender>
   <signal>clicked()</signal>
   <receiver>LoginWidget</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>447</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnWinClose_2</sender>
   <signal>clicked()</signal>
   <receiver>LoginWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>118</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>431</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnWinMin</sender>
   <signal>clicked()</signal>
   <receiver>LoginWidget</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>358</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnWinClose</sender>
   <signal>clicked()</signal>
   <receiver>LoginWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>388</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>468</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
